core/src/main/java/org/apache/calcite/sql/validate/SqlValidator.java

```diff
@@ -655,6 +655,15 @@ void setValidatedNodeType(
    */
   SqlValidatorScope getOrderScope(SqlSelect select);
 
+  /**
+   * Returns the scope that should be used when converting the create table
```

Comment:
This is just getting the create table scope right? "the scope that should be used" seems ambiguous, can we update to "Returns scope of the create table" if that is correct?

Reply:
Yes, and done.